deploy/databreachdetector/delete.sh

```
@@ -0,0 +1,30 @@
+# Uninstalls print service
+## Usage: ./delete.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+function deleting_databreachdetector() {
+  NS=databreachdetector
+  while true; do
+      read -p "Are you sure you want to delete print helm chart?(Y/n) " yn
+      if [ $yn = "Y" ]
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)